recipes/brass/meta.yaml

```diff
@@ -0,0 +1,37 @@
+build:
+  number: 2
+  skip: True # [osx]
+
+package:
+    name: brass 
+    version: 5.1.6
+source:
+    fn: 5.1.6.tar.gz
+    url: https://github.com/cancerit/BRASS/archive/v5.1.6.tar.gz
+    md5: bb33dec7048cd1634845f143ed706e3a
+    patches:
+       - makefile.patch
+       - cansam_headers.patch
+
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - perl-pcap
+    - perl-grass
+    - cansam
+    - boost
```

Comment:
You want to pin the boost version, please the the guideline section about this one.
And have you forgotten boost as runtime dep?